app/Services/Campaign/ExportService.php

```diff
@@ -113,6 +115,26 @@ public function filesize(): int
         return $this->filesize;
     }
 
+
+    protected function campaignModules(): self
+    {
+        $modules = [];
+        foreach ($this->campaign->modules() as $name => $active) {
+            $module = ['enabled' => $active];
+            if (!in_array($name, ['dice_rolls', 'conversations', 'bookmarks', 'inventories', 'entity_attributes', 'assets'])) {
```

Comment:
This feels like a pain to maintain and would be easier supported by a try/cache so we never have tot hink about it